pachyderm/values.yaml

```diff
@@ -75,9 +79,13 @@ etcd:
   storageClass: ""
   # storageSize specifies the size of the volume to use for etcd.
   storageSize: "100Gi"
+  # podLabels specifies labels to add to the etcd pod.
+  podLabels: {}
   service:
     # annotations specifies annotations to add to the etcd service.
     annotations: {}
+    # labels specifies labels to add to the etcd service.
+    labels: {}
```

Comment:
Labels typically follow annotations so I did the same here. To maintain consistently I then ensured that it preceded type in all other services.